vue-frontend/src/components/partials/HeaderV2.vue

```diff
@@ -0,0 +1,214 @@
+  data() {
+    return {
+      headerLogoImage: headerLogoImage,
+      careerURL: "/jobs",
+      contactURL: "/contact",
+      blogsURL: Config.BLOG_URL,
+      navbarSticky: false,
+      navbarAnimation: false,
+      navContainerHeight: 0,
+      lastScrollY: 0,
+      activeCareerPath: false,
+      activeContactPath: false,
+    };
+  },
+  components: {},
+  mounted() {
+    let currentRoutePath = this.$router.currentRoute._value.path;
+
+    this.activeCareerPath = currentRoutePath == this.careerURL;
+
+    this.activeContactPath = currentRoutePath == this.contactURL;
+
+    window.addEventListener("scroll", this.handleScroll);
+    this.navContainerHeight = this.$refs.mainHeader.clientHeight + 30;
+  },
+  unmounted() {
+    window.removeEventListener("scroll", this.handleScroll);
+  },
+  computed: {},
+  methods: {
+    handleScroll() {
+      let wasSticky = this.navbarSticky;
+      this.navbarSticky = window.scrollY > 30;
+
+      // If scroll diff is large, we show navbar with animation
+      let diff = window.scrollY - this.lastScrollY;
+      if (this.navbarSticky != wasSticky && diff > 15) {
+        this.navbarAnimation = true;
+      }
+      if (!this.navbarSticky) {
+        this.navbarAnimation = false;
+      }
+      this.lastScrollY = window.scrollY;
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.nav-container {
+  background: #fff;
+  position: relative;
+}
+
+.navbar {
+  padding: 10px 2%;
+  width: 100%;
+  background: #fff;
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  z-index: 1;
+  transition: all 0.6s ease-in-out;
+}
+
+.navbar-sticky {
+  position: fixed;
+  left: unset;
+  bottom: unset;
+  box-shadow: 0 13px 35px -12px rgba(35, 35, 35, 0.15);
+}
+
+a {
+  color: #000;
+}
+
+.navbar-animation {
+  animation: menu_sticky 0.6s ease-in-out;
+}
+
+.bottom-border::after {
+  content: "";
+  display: block;
+  position: relative;
+  width: 50.4%;
+  border-top: 3px solid black;
+}
+
+@keyframes menu_sticky {
+  0% {
+    top: -120px;
+    opacity: 0;
+  }
+
+  100% {
+    top: 0;
+    opacity: 1;
+  }
+}
+
+.navbar-nav {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+}
+
+.navbar-nav .nav-item-margin .nav-link {
+  margin: 0 20px 0 0;
+  color: rgb(0, 0, 0);
+}
+
+.header-logo-image {
+  width: 205px;
+  height: 38.5px;
+}
+
+@include media-breakpoint-up(md) {
+  .navbar-nav .nav-item-margin .nav-link {
+    margin: 0 20px 0 0;
+  }
+
+  .navbar {
+    padding: 20px 0;
+  }
+  .col-md {
+    flex-direction: column;
+  }
+}
+
+@include media-breakpoint-up(lg) {
+  .round-btn {
+    background-color: #000;
+    border-radius: 25px;
+    padding: 10px 25px;
+    color: #fff;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  }
+  .round-btn:hover > a {
+    color: black;
+  }
+  .round-btn:hover {
+    background-color: white;
+    color: black;
+  }
+  .anchor {
+    color: white;
+  }
+  .col-md {
```

Comment:
Why bootstrap class define over here

Reply:
no its a userdefined. should i have to change the naming convention ?

Reply:
No, it totally removes and refers to the `underline-text` css class that we used in the All Ready Exciting website.